### PR TITLE
Rax username and api key

### PR DESCRIPTION
**Pull request by grahamrhay**
Adds support for the RAX_USERNAME & RAX_API_KEY env vars (same as the rax module). Useful when running on a build agent.

### CODE REVIEW

**Comment by mpdehaan** [comment]
Thanks!

(sidenote for us: the DOCUMENTATION section is not something that is read from inventory scripts, we should move any docs to the INI file)

File: plugins/inventory/rax.py [hunk 1]
```diff
@@ -269,6 +249,38 @@ def setup():
 
     return regions
 
+def set_credentials(region):
+    username = os.getenv('RAX_USERNAME', None)
+    if username:
+        api_key = os.getenv('RAX_API_KEY', None)
+        pyrax.set_credentials(username, api_key)
```
Comment (sivel) [inline plugins/inventory/rax.py]: To remain consistent with the other auth calls (`pyrax.keyring_auth` and `pyrax.set_credential_file`), you should also pass `region=region` to `pyrax_set_credentials` such as:

```
pyrax.set_credentials(username, api_key, region=region)
```

**Comment by amenonsen** [comment]
@grahamrhay any plans to update this to apply to current devel?

**Comment by amenonsen** [comment]
Since this PR hasn't been updated for months, it should probably be closed now. However, @sivel is listed as an author of inventory/rax.py, so I'll leave it to him to opine on whether this feature is worth trying to resurrect.

**Comment by grahamrhay** [comment]
sorry, missed that notification! I'll rebase, and re-open

**Comment by amenonsen** [comment]
Thanks, @grahamray. Since you've said you intend to rebase and resubmit, I'll close this (outdated) PR now.

**Pull request closed without merging**